inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/config/Swagger2Config.java

```diff
@@ -38,6 +42,9 @@ public class Swagger2Config {
     public Docket createRestApi() {
         return new Docket(DocumentationType.SWAGGER_2)
                 .apiInfo(apiInfo())
+                .directModelSubstitute(LocalDate.class, String.class)
```

Comment:
Thanks for your contribution.

Firstly, please create an issue here: <https://github.com/apache/inlong/issues/new/choose>,

Secondly, could you please explain why add those configs?